File: src/TmrOverlay.App/Overlays/Relative/RelativeOverlayDefinition.cs
using TmrOverlay.Core.Overlays;

namespace TmrOverlay.App.Overlays.Relative;

internal static class RelativeOverlayDefinition
{
    public static OverlayDefinition Definition { get; } = new(
        Id: "relative",
        DisplayName: "Relative",
        DefaultWidth: 360,
        DefaultHeight: 373,
        Options:
        [
            OverlaySettingsOptionDescriptor.Integer(
                OverlayOptionKeys.RelativeCarsEachSide,
                "Cars each side",
                0,
                8,
                defaultValue: 5)
        ],
        FadeWhenLiveTelemetryUnavailable: true);
}

File: local-mac/TmrOverlayMac/Sources/TmrOverlayMac/Overlays/Relative/RelativeOverlayDefinition.swift
import AppKit

enum RelativeOverlayDefinition {
    static let definition = OverlayDefinition(
        id: "relative",
        displayName: "Relative",
        defaultSize: NSSize(width: 520, height: 360),
        fadeWhenLiveTelemetryUnavailable: true
    )
}
